server/config.ts

```diff
@@ -95,7 +95,8 @@ export default {
     },
   },
   domain: get('INGRESS_URL', 'http://localhost:3000', requiredInProduction),
-  gtmContainerId: get('GOOGLE_TAG_MANAGER_CONTAINER_ID', null),
```

Comment:
My original work to enable GA wasnt too far out to be honest. We now have the concept of 2 container IDs based on 2 env variables (which may be null - the original one has always been null :) )
The change here is basically to create 2 config values for the 2 different container IDs from the env vars which may or may not have been setup from the kube secrets